docs/src/prototypes/CopyToClipboard/NotificationProvider.tsx

```diff
@@ -28,9 +35,21 @@ export const NotificationProvider: React.FC = props => {
 
   return (
     <>
-      <Portal open={!!notification}>
-        <Notification>{notification}</Notification>
-      </Portal>
+      {!!targetRef ? (
+        !!notification && (
+          <Tooltip
```

Comment:
The following props are missing, yet they are used in `CopyToClipboard.tsx`'s `Tooltip`: align, pointing, position.